### ISSUE TITLE
Allow bosh addons on specific networks

### ISSUE THREAD

**Issue by schmidtsv** [cloudfoundry/bosh#1625]
Hi, the addon structure for bosh allows you to specify deployments to which an addon should be installed like this: 

```yaml
include:
  deployments: [dep1, dep2]
  jobs:
  - name: redis
    release: redis-release
  stemcell:
    - os: ubuntu-trusty
```

On our vsphere-staging and concourse environment we are currently doing some stuff with the networking release (set gateway to public, propagate some routes, close port 22 on 0.0.0.0) which is currently done in the specific deployment manifests. We would be very interested, if the include rules would also allow you to define a inclusion filter, that will install this job on all deployments that are in a specific network like this:

```yaml
include:
  networks: [elastic]
```

**Comment by dpb587-pivotal** [comment]
This is being implemented via PR #1653

**Comment by cppforlife** [comment]
PR was merged. will be in v262.
